api-tests/src/test/java/org/openmrs/module/reporting/cohort/definition/evaluator/EncounterCohortDefinitionEvaluatorTest.java

```diff
@@ -260,17 +260,15 @@ public void evaluate_shouldFindPatientsWithEncountersOnTheOnOrBeforeDateIfPassed
 	@Verifies(value = "should find patients with encounters created on the specified date if passed in time is at midnight", method = "evaluate(CohortDefinition,EvaluationContext)")
 	public void evaluate_shouldFindPatientsWithEncountersCreatedOnTheSpecifiedDateIfPassedInTimeIsAtMidnight()
 	    throws Exception {
+		executeDataSet(XML_DATASET_PATH + "ReportTestDataset-encounter-before-midnight.xml");
```

Comment:
See my comments on ReportTestDataset-encounter-before-midnight.xml for these changes